src/Components/List/List.jsx

```diff
@@ -130,12 +147,29 @@ export default function List({ items, token }) {
                     onChange={() => purchaseItem(item, token)}
                     checked={checked}
                     disabled={checked}
+                    aria-label={`Mark as purchased`}
```

Comment:
Thanks @mxmason for this suggestion! We tested this out with Voiceover, and decided to forego adding the item name to the aria-label due to the way the elements are set up (item name is a label for the checkbox input). With the item name in the aria-label, the checkbox read as "Mark as purchased bananas bananas". Could we have tackled this in a different way? It seemed redundant, but is over-describing the checkbox better than under-describing in this case?

Reply:
Ah, I understand your concern! It's true that my suggestion puts the name of the item in the code twice, and it could seem repetitive, but: with your code as it is now, someone who is using the `Tab` key with Voiceover would _skip over_ the text node and never hear it. e.g.,

1. I am focused on the search field. I press `Tab`
2. Focus moves to the first item. I hear `mark as purchased` because that is what you labeled the checkbox
3. I press `Tab` again
4. Focus moves to the next checkbox. I hear `mark as purchased`. I now do not know what either of these items were

In this scenario, I as a VoiceOver user still have the _option_ to manually read all the text nodes; I _could_ hear the name of the items, but it also could get lost. Try what I am talking about on your own machine!

Reply:
Thanks EJ! This is super helpful to know.